google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -256,9 +256,10 @@ def __init__(
             self._region = connection_string_split[1]
             self._instance = connection_string_split[2]
         else:
-            raise CloudSQLConnectionError(
-                "Arg instance_connection_string must be in "
-                + "format: project:region:instance."
+            raise CloudSQLConnectionStringError(
```

Comment:
Does it make sense to use a typed error here, or would it be better to throw a ValueError?